src/index.html

```diff
@@ -705,12 +705,12 @@ <h5 class="modal-title" id="methodModalLabel">Explanation of methods</h5>
       </div>
       <div class="modal-body">
         <a data-bind="visible: showCross"/></a>
-        <h2>Cross-sectional</h2>
+        <h2>Calendar year</h2>
```

Comment:
These blurbs will also be updated pending us receiving new text from Katy, there's a separate ticket for that.